app/views/layouts/base.html.erb

```diff
@@ -55,9 +55,7 @@
       ) %>
   <meta content="#ffffff" name="theme-color" />
 
-  <% if BrowserSupport.supported?(request.user_agent) && IdentityConfig.store.newrelic_browser_key.present? && IdentityConfig.store.newrelic_browser_app_id.present? %>
-    <%= render 'shared/newrelic/browser_instrumentation' %>
-  <% end %>
+  <%= render_javascript_pack_once_tags('track-errors-prelude') if BrowserSupport.supported?(request.user_agent) %>
```

Comment:
`BrowserSupport.supported?` is cached and I assume the cache lookup would be reasonably fast, but we could also consider storing the result in a variable somewhere.

Reply:
Decided to inline this script for all browsers.